readme.md

```diff
@@ -59,6 +59,13 @@ Default: `false`
 
 Force kill the process.
 
+##### forceTimeout
```

Comment:
This makes it look like it's a timeout for the `force` option, which it's not. Yes, the description clarifies this, but would be nice to have a different option name instead.